spark/util.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source /usr/lib/hustler/bin/qubole-bash-lib.sh
+
+##
+# Start Spark History Server
+#
+function start_history_server() {
+   is_master=$(nodeinfo is_master)
+    if [[ "$is_master" == "1" ]]; then
+        monit start sparkhistoryserver
+    fi
+}
+
+##
+# Stop Spark History Server
+#
+function stop_history_server() {
+   is_master=$(nodeinfo is_master)
+    if [[ "$is_master" == "1" ]]; then
+        monit stop sparkhistoryserver
+    fi
+}
+
+##
+# Restart Spark History Server
+#
+function restart_history_server() {
+    stop_history_server && sleep 2 && start_history_server
```

Comment:
We can use **monit restart sparkhistoryserver** here. By using the restart we don't have to use sleep.

Reply:
Do you have a specific sequence defined for restart in the monitrc? Otherwise restart via monit is unreliable because it doesn't actually wait for the service to be stopped.

Reply:
We don't have any sequence in the restart in the monitrc, If  restart via monit is unreliable than we can go with the approach given in the PR.